### ISSUE TITLE
PMID:19826040

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3663]
Hi @RLovering  - @pgaudet  and I are tidying the cell cycle checkpoints.

We will remove the 'regulation of checkpoint' terms, because the cell cycle signalling *is* the regulation.

Thare are only a few annotations to  regulation of the checkpoint and 2 are from
PMID:19826040

URS000039ED8D_9606 | Homo sapiens (human) hsa-miR-21-5p |   | regulation of signal transduction involved in mitotic G2 DNA damage checkpoint | happens during cellular response to gamma radiation | BHF-UCL | Homo sapiens | IMP

URS000039ED8D_9606 | Homo sapiens (human) hsa-miR-21-5p |   | negative regulation of G1/S transition of mitotic cell cycle | happens during cellular response to stress | BHF-UCL | Homo sapiens | IMP

These seem to be incorrect  (phenotypes) because the checkpoints are post-transcriptional.
They operate through signalling pathways beginning way upstream of cdc25 (although something that interfered with the expression of cdc25 would result in a  cell cycle block because it would cause problems  in cell cycle processes more generally, and the checkpoints would not work if cdc25   levels were compromised.

Taken together, these results indicate that miR-21 induction inhibits the G1-S transition by suppressing Cdc25A expression.
The above results suggest that miR-2-mediated down-regulation of Cdc25A contributes to the activation of the G2-M checkpoint following radiation.

**Comment by RLovering** [comment]
Hi Val

thanks for explaining this. do you want me to just delete these annotations - as you are removing the regulation terms. Or would you be able to suggest the annotations I should change these too. Sorry I just don't have time to read this paper along with all my teaching and you are just so good at this area, whereas I actively avoid it ;(

Best

Ruth

**Comment by ValWood** [comment]
Hi Ruth,

I think the annotation is captured by

URS000039ED8D_9606 | Homo sapiens (human) hsa-miR-21-5p |   | mRNA binding involved in posttranscriptional gene silencing | has input UniProtKB:P30304 | BHF-UCL | Homo sapiens | IDA |   |   | miRNA |   | PMID:19826040 | 20151015


I would also remove

URS000039ED8D_9606 | Homo sapiens (human) hsa-miR-21-5p |   | negative regulation of G1/S transition of mitotic cell cycle | happens during cellular response to stress | BHF-UCL | Homo sapiens | IMP |   |   | miRNA |   | PMID:19826040 | 20151015


You have
regulates signal transduction involved in mitotic G2 DNA damage checkpoint
in an extension too, so that would need to be removed.

The experiments are done in cancer cells. The target was identified from a number of potential from a microarray experiment, although confirmed lots of genes were unregulated. The expression level difference were not huge. Checkpoint regulation is post translational and originates upstream of cdc25, so engineering a change in the level  of the expression of cdc25 can't be part of checkpoint signalling _per se_

In summary  would not take too much from this paper about the normal processes in relation to cell cycle.  You could only really say "causally_upstream_or_within" "regulation of mitotic cell cycle" from the information here. If it is doing this. 
(I can totally see why these annotations  would be made from this paper  from the wording, though without knowing how the signalling begins) 

I did not know anything about microRNA-21, but is has 13625 publications!

cheers,

Val

**Comment by RLovering** [comment]
great thanks
annotations removed or updated
